Packs/Core/Triggers/Trigger_-_A_source_host_created_uncommon_remote_scheduled_task.json

```diff
@@ -0,0 +1,19 @@
+{
+  "trigger_id": "df44be7092df167c73c5fa4eecc8977b",
+  "playbook_id": "A source host created uncommon remote scheduled task",
```

Comment:
don't forget to change after name change

Reply:
Done.